ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/CalciteSemanticException.java

```diff
@@ -34,7 +34,7 @@ public enum UnsupportedFeature {
     Having_clause_without_any_groupby, Invalid_column_reference, Invalid_decimal,
     Less_than_equal_greater_than, Others, Same_name_in_multiple_expressions,
     Schema_less_table, Select_alias_in_having_clause, Select_transform, Subquery,
-    Table_sample_clauses, UDTF, Union_type, Unique_join,
+    Table_sample_clauses, UDTF, Unique_join,
```

Comment:
Clarification: The Union_type is not need anymore?

Reply:
This is an internal enumeration that keeps track of `UnsupportedFeature`s in CBO. Since now UNIONTYPE is supported in CBO the enum entry is not necessary.